src/scss/component/_container.scss

```diff
@@ -79,12 +79,22 @@
 		@include oTypographyMargin($top: 3);
 		@include oTypographyPadding($bottom: 2);
 		border-bottom: 1px solid oColorsGetPaletteColor('grey-10');
+		position: relative;
 
 		.demo__code,
 		.demo__live {
 			width: 100%;
 			border: 0;
 			appearance: none;
+
+		}
+
+		button.select-html {
+			@include oButtons();
+			@include oButtonsCustomTheme($theme: $demo-custom-buttons);
```

Comment:
A little pedantic for the registry maybe, but if this button theme is output already could we share a button class and reposition it based on its context or some other selector?